tfhe/src/c_api/high_level_api/integers.rs

```diff
@@ -659,3 +659,112 @@ pub unsafe extern "C" fn compact_fhe_int256_list_try_encrypt_with_compact_public
 }
 
 define_all_cast_into_for_integer_type!(FheBool);
+
+macro_rules! impl_oprf_for_uint {
+    (
+        name: $name:ident
+    ) => {
+
+        ::paste::paste! {
+            #[no_mangle]
+            pub unsafe extern "C" fn [<par_generate_oblivious_pseudo_random_ $name:snake>](
+                out_result: *mut *mut $name,
+                seed_low_bytes: u64,
+                seed_high_bytes: u64,
+                random_bits_count: u64,
+            ) -> c_int {
+
+                $crate::c_api::utils::catch_panic(|| {
+                    let seed_low_bytes: u128 = seed_low_bytes.into();
+                    let seed_high_bytes: u128 = seed_high_bytes.into();
+                    let seed = concrete_csprng::seeders::Seed((seed_high_bytes << 64) | seed_low_bytes);
```

Comment:
we should be able to use crate::Seed I believe ? as we re-export it at the HL level I think